runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineRunner.java

```diff
@@ -113,12 +113,7 @@ public PortablePipelineResult run(RunnerApi.Pipeline pipeline, JobInfo jobInfo)
             ? trimmedPipeline
             : GreedyPipelineFuser.fuse(trimmedPipeline).toPipeline();
 
-    // File staging.
     prepareFilesToStage(pipelineOptions);
-    LOG.info(
```

Comment:
Oh how did I miss this one. Thanks @ibzib for the fix !

Reply:
Thank you! and thanks for enabling the nullness checker as well